src/System.Runtime.Extensions/src/System/IO/Path.Windows.cs

```diff
@@ -93,6 +84,15 @@ public static string GetFullPath(string path)
 
             return fullPath;
         }
+        
+        public static char[] GetInvalidFileNameChars() => new char[]
```

Comment:
Yes, please move it back up.  Part of my concern with changes like this is that they run the risk of introducing regressions, e.g. if one of the characters in this array was changed.  Moving stuff around in the file makes it very difficult to see in the diff view if there are actually meaningful changes, so it's best not to incur unnecessary changes.  I'm still not convinced of the value of this change, but it has a higher chance of getting merged if it's painfully obvious that it's not going to cause problems.